tests/wsgi/tests/test_util.py

```diff
@@ -88,6 +88,42 @@ def test_build_precache_map():
     precache_map = build_precache_map(config)
     assert precache_map == {'enwiki': {'edit': {'revid', 'goodfaith'}}}
 
+def test_build_event_set():
```

Comment:
Looks like the flake8 check is failing on TravisCI here:
```
/ores/tests/wsgi/tests/test_util.py:91:1: E302 expected 2 blank lines, found 1
```

Add another line before this test function and you should be good to go.